### PR TITLE
rip: confetti 

### PR DESCRIPTION
**Pull request by stack72**
it has served us well but it's the time to remove it

### CODE REVIEW

**Review by nickgerace — DISMISSED** [review]
https://www.youtube.com/watch?v=lpZiPZwwXhM

**Comment by wendybujalski** [comment]
😭

**Review by wendybujalski — APPROVED** [review]

File: app/web/src/App.vue [hunk 1]
```diff
@@ -17,12 +17,7 @@
     <template v-else>
       <CachedAppNotification />
       <RouterView :key="selectedWorkspace?.pk" />
-      <Teleport to="body">
-        <canvas
-          id="confetti"
-          class="fixed w-full h-full top-0 left-0 pointer-events-none z-100"
-        ></canvas>
-      </Teleport>
+      <Teleport to="body" />
```
Comment (wendybujalski) [inline app/web/src/App.vue:20]: Actually I think this line should just be removed, no?

Comment (wendybujalski) [reply app/web/src/App.vue:20]: (line 20, the <Teleport /> that we are no longer using

**Review by wendybujalski — CHANGES_REQUESTED** [review]
remove the unnecessary `Teleport` on line 20 of App.vue and this will be good to go

**Comment by stack72** [comment]
> app/web/src/App.vue

done!!

**Review by wendybujalski — APPROVED** [review]
😭

**Pull request merged**